concepts/Common/Security Considerations/20 HTML Encoding/40 Encoding in Templates.md

```diff
@@ -0,0 +1,34 @@
+Angular, Vue, and React always encode values interpolated in templates. Without these frameworks, use the `encodeHtml` utility method to encode the interpolated values:
+
+    <!-- tab: JavaScript -->
+    $(function() {
+        const encodeHtml = DevExpress.utils.string.encodeHtml;
+        $("#tabs").dxTabs({
+            dataSource: tabs,
+            width: 600,
+            itemTemplate: function (itemData) {
+                return encodeHtml(itemData.content);
+            }
+        });
+    });
+
+    const tabs = [{     
+        id: 0,
+        content: "<img src=1 onerror=alert('XSS') \/>" 
+    }, { 
+        id: 1,
+        content: "<script>alert('XSS')<\/script>" 
+    }, { 
+        id: 2, 
+        content: "Tab content" 
+    }];
+
+<a href="https://codepen.io/romantsukanov/pen/vYKjzQw?editors=1010" target="_blank">View on CodePen</a>
+
+Inserting unencoded content can open your application to XSS attacks:
+
+![DevExtreme Tabs with disabled HTML encoding](/images/UiWidgets/tabs-itemTemplate-unsafe-html.png)
+
+With encoding, the content is interpreted as text and simply displayed:
```

Comment:
Is it technically correct to say the following instead?

The encoded content is interpreted and displayed as text: